app/javascript/views/super_admin/notifications/edit.js

```diff
@@ -1,5 +1,39 @@
+<<<<<<< HEAD
+import { Tinymce } from '../../../utils/tinymce';
+// add the info on selecting the check from notification suitable
+=======
 import { Tinymce } from '../../../utils/tinymce.js.erb';
 
+//add the info on selecting the check from notification suitable
+>>>>>>> 8cebbdfe52128dbb2af7c3e1f7f1407be62795c4
+import { paginableSelector } from '../../../utils/paginable';
+import * as notifier from '../../../utils/notificationHelper';
+
+
 $(() => {
   Tinymce.init({ selector: '.notification-text', forced_root_block: '' });
+
+
+  $(paginableSelector).on('click, change', '.set_active_notification input[type="checkbox"]', (e) => {
+    const form = $(e.target).closest('form');
+    form.submit();
+  });
+
+
+  $(paginableSelector).on('ajax:success', '.set_active_notification', (e, data) => {
+    // const form = $(e.target);
```

Comment:
If this line isn't needed please remove instead of commenting out

Reply:
The webpacker was complaining about a variable that was not being used. I comment it out because I wasn't sure what is meant to do and couldn't see it being used anywhere

Reply:
Yeah, I'd just remove it instead of leaving the comment in.
If somebody wants to see what used to be there, they can always go through the github history, and it keeps the codebase a bit cleaner